src/NuGet.Clients/NuGet.Tools/NuGet.Tools.csproj

```diff
@@ -334,13 +344,44 @@
     </None>
   </ItemGroup>
   <ItemGroup />
-  <Target Name="AfterBuild">
+  <Target Name="AfterBuild" DependsOnTargets="AfterBuild14;AfterBuild15;AfterBuildIns15"/>
+  <Target Name="AfterBuild14"  Condition="'$(VisualStudioVersion)' == '14.0'">
+    <PropertyGroup>
+      <PublishDestination>$(ArtifactRoot)\$(VsixOutputDirName)</PublishDestination>
+    </PropertyGroup>
+    <Message Text="Publishing VS14 VSIX package files..." Importance="high" />
+    <Message Text="$(TargetVsixContainer) -&gt; $(PublishDestination)" Importance="high" />
+    <Copy SourceFiles="$(TargetVsixContainer)" DestinationFolder="$(PublishDestination)" OverwriteReadOnlyFiles="true" />
+    <ItemGroup>
+      <NuGetSymbolFiles Include="$(OutputPath)NuGet*.pdb" />
+    </ItemGroup>
+    <Message Text="Copying symbols file '%(NuGetSymbolFiles.Identity)'" Condition=" '@(NuGetSymbolFiles)' != '' " />
+    <Copy SourceFiles="@(NuGetSymbolFiles)" DestinationFolder="$(PublishDestination)\symbols" OverwriteReadOnlyFiles="true" />
+  </Target>
+  <Target Name="AfterBuild15" Condition="'$(VisualStudioVersion)' == '15.0'">
     <PropertyGroup>
       <PublishDestination>$(ArtifactRoot)\$(VsixOutputDirName)</PublishDestination>
     </PropertyGroup>
-    <Message Text="Publishing VSIX package files..." Importance="high" />
+    <Message Text="Publishing VS15 VSIX package files..." Importance="high" />
     <Message Text="$(TargetVsixContainer) -&gt; $(PublishDestination)" Importance="high" />
     <Copy SourceFiles="$(TargetVsixContainer)" DestinationFolder="$(PublishDestination)" OverwriteReadOnlyFiles="true" />
+    <Message Text="$(ArtifactRoot)\NuGet.Tools\15.0\$(Configuration)\Microsoft.VisualStudio.NuGet.Core.json -&gt; $(PublishDestination)" Importance="high" />
```

Comment:
Path to the generated file should be `$(OutputPath)Microsoft.VisualStudio.NuGet.Core.json`